kotlin/src/com/google/idea/blaze/kotlin/run/producers/BlazeKotlinRunConfigurationProducer.java

```diff
@@ -152,6 +152,6 @@ private static FilteredTargetMap computeTargetMap(Project project, BlazeProjectD
         project,
         projectData.artifactLocationDecoder,
         projectData.targetMap,
-        (target) -> target.kind == Kind.KT_JVM_BINARY && target.isPlainTarget());
+        (target) -> (target.kind == Kind.KT_JVM_BINARY || target.kind == Kind.JAVA_BINARY) && target.isPlainTarget());
```

Comment:
```
KotlinRunConfigurationProducer.Companion.getEntryPointContainer(location.getPsiElement());
```
above should return `null` for `java_binary`s without a kotlin main method, correct?

Just making sure this wouldn't show up for regular `java_binary`s.

Reply:
correct, and thanks !